pkg/flowaggregator/flowaggregator.go

```diff
@@ -294,6 +299,27 @@ func (fa *flowAggregator) InitAggregationProcess() error {
 	return err
 }
 
+func (fa *flowAggregator) createAndSendTemplate(isRecordIPv6, isOriginExporterIPv6 bool) (uint16, error) {
+	templateID := fa.exportingProcess.NewTemplateID()
+	recordIPFamily := "IPv4"
+	exporterIPFamily := "IPv4"
+	if isRecordIPv6 {
+		recordIPFamily = "IPv6"
+	}
+	if isOriginExporterIPv6 {
+		exporterIPFamily = "IPv6"
+	}
+	bytesSent, err := fa.sendTemplateSet(isRecordIPv6, isOriginExporterIPv6)
+	if err != nil {
+		fa.exportingProcess.CloseConnToCollector()
+		fa.exportingProcess = nil
+		fa.set.ResetSet()
+		return 0, fmt.Errorf("sending %s template set with %s original exporter ip failed, err: %v", recordIPFamily, exporterIPFamily, err)
+	}
+	klog.V(2).Infof("Initialized exporting process and sent %d bytes size of %s template set with %s original exporter ip", bytesSent, recordIPFamily, exporterIPFamily)
```

Comment:
please use structured logging (`klog.InfoS` / `klog.ErrorS`) for all new log messages